lib/matplotlib/ticker.py

```diff
@@ -81,6 +81,19 @@
 
 The default minor locator is `NullLocator`, i.e., no minor ticks on by default.
 
+.. note::
+    `Locator` instances should not be used with more than one
+    `~matplotlib.axis.Axis` or `~matplotlib.axis.Axes`. So instead of::
+
+        locator = matplotlib.ticker.MultipleLocator(5)
+        ax.xaxis.set_major_locator(locator)
+        ax2.xaxis.set_major_locator(locator)
```

Comment:
- I would leave out the qualification `matplotlib.ticker`. This makes it more verbose, and you actually would usually not `import matplotlib.ticker`. It's viable to assume `from matplotlib.ticker import MultipleLocator`. Moreover, in the above text we use the class names a lot, so `MultipleLocator` without qualification should be clear.
- Using `xaxis` and `yaxis` instead of two different Axes is a bit simpler and serves the same purpose.


```suggestion
        locator = MultipleLocator(5)
        ax.xaxis.set_major_locator(locator)
        ax.yaxis.set_major_locator(locator)
```

Reply:
But, in my opinion, the predominant use is sharing e.g. date-based locators across `Axes` instances for the x/y axis. That's the case that prompted this.

Reply:
Fair enough.

What about the first bullet point?

Reply:
Missed that the first time--seems fine. Done.